### ISSUE TITLE
Canada’s Open Education Initiatives A11Y: Table needs table headers and caption

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#155]
**Describe the bug**
The contact info table does not have proper table markup. I think this is a non-dev task.

**To Reproduce**
See this page: https://helga.bccampus.ca/wp/opentextbooks/what-is-open-education/canadas-open-education-initiatives/

**Expected behavior**
Column headers should be set as headers with a "column" scope assigned. There should also be a caption for the table.

**Screenshots**
![Screenshot_2019-03-25 Canada’s Open Education Initiatives – open UAT site](https://user-images.githubusercontent.com/23706762/54933166-5b692d00-4ed9-11e9-91eb-65adfa875f2f.png)

**Comment by ehbeattie** [comment]
I'm pretty sure I just addressed this one. Right?

**Comment by josieg** [comment]
Lol yes. I forgot I created an issue for it. Close!
